CHANGELOG.md

```diff
@@ -5,15 +5,12 @@ module, now available on the PowerShell Gallery. Please switch to the new module
 functions from Carbon, a warning message will be written. These function will be removed in the next major version of
 Carbon.
 
-
-# 2.12.0 
-
 ## Added suport for PowerShell Core

```

Comment:
Remove this heading and all empty lines between the two "sections". Also, looks like the last item doesn't break at character/column 120. Please add if missing.

```markdown
* The `Install-CMsi`, `Get-CMsi`, and `Get-CProgramInstallInfo` functions have moved to a new `Carbon.Windows.Installer`
module, now available on the PowerShell Gallery. Please switch to the new module and update usages. If you use these
functions from Carbon, a warning message will be written. These function will be removed in the next major version of
Carbon.
* Carbon now works under PowerShell 7.
* Fixed: Get-CPermission fails to grant permissions on certificates in PowerShell 6+ and on certificates that .NET
Framework load as RSA cryptographic next generation keys, which don't have an API for setting private key permissions.
```